flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+	"only-arches": ["x86_64"]
```

Comment:
this is disappointing.

Reply:
?

Reply:
that ypu chose to just ignore other arch. This is disappointing.